cmd/tether/attach.go

```diff
@@ -584,3 +586,50 @@ type winsize struct {
 	wsXpixel uint16
 	wsYpixel uint16
 }
+
+type stdinReader struct {
+	reader io.ReadCloser
+}
+
+func newStdinReader(channel ssh.Channel) *stdinReader {
+	r, w := io.Pipe()
+	s := &stdinReader{
+		reader: r,
+	}
+
+	go func() {
+		buf := make([]byte, 512)
```

Comment:
in most cases when you will not notice significant benefits of the increased buffer size is around 8-64k, which means that even maybe 2k is small and 4k is better.